pyccl/ccl_power.i

```diff
@@ -12,30 +12,6 @@
 
 %include "../include/ccl_power.h"
 
-/* The python code here will be executed before all of the functions that
-   follow this directive. */
-%feature("pythonprepend") %{
-    if numpy.shape(k) != (nout,):
-        raise CCLError("Input shape for `k` must match `(nout,)`!")
-%}
-
-%inline %{
-void linear_matter_power_vec(ccl_cosmology * cosmo, double a, double* k, int nk,
```

Comment:
Note to self: figure out what replaces this.

Reply:
By this: https://github.com/LSSTDESC/CCL/blob/6dcd05f7c3990dd7706f58d6996b1c86e3965639/pyccl/power.py#L62